Sources/Localization/en.lproj/Localizable.strings

```diff
@@ -514,3 +514,81 @@
 "ZAR"="Rand";
 "ZMW"="Zambian Kwacha";
 "ZWL"="Zimbabwe Dollar";
+
+// Transaction Type Content
+"account_closure_fee" = "Account Closure Fee";
+"account_closure" = "Account Closure";
+"account_unload" = "Funds Unloaded";
+"adjustment" = "Balance Adjustment";
+"annual_fee_refund" = "Annual Fee Refund";
+"annual_fee" = "Annual Fee";
+"bank_account_transfer_fee" = "Bank Account Return Fee";
+"bank_account_transfer_return_fee" = "Bank Account Return Fee";
+"bank_account_transfer_return" = "Bank Account Return";
+"card_activation_fee_waiver" = "Card Activation Fee Waiver";
+"card_activation_fee" = "Card Activation Fee";
+"card_fee" = "Card Fee";
+"customer_service_fee_refund" = "Customer Service Fee Refund";
+"customer_service_fee" = "Customer Service Fee";
+"deposit" = "Funds Deposit";
+"donation_fee" = "Donation Fee";
+"donation_return" = "Donation Return";
+"donation" = "Donation";
+"dormant_user_fee_refund" = "Domant User Account Fee Refund";
+"dormant_user_fee" = "Domant User Account Fee";
+"expedited_shipping_fee" = "Expedited Shipping Fee";
+"foreign_exchange" = "Foreign Exchange";
+"generic_fee_refund" = "Generic Fee Refund";
+"manual_adjustment" = "Adjustment";
+"manual_transfer_to_prepaid_card" = "Card Load";
+"merchant_payment_fee" = "Merchant Payment Fee";
+"merchant_payment_refund" = "Merchant Payment Refund";
+"merchant_payment_return" = "Merchant Payment Return";
+"merchant_payment" = "Merchant Payment";
+"moneygram_transfer_return" = "MoneyGram Return";
+"monthly_fee_refund" = "Monthly Fee Refund";
+"monthly_fee" = "Monthly Fee";
+"paper_check_fee" = "Paper Check Fee";
+"paper_check_refund" = "Paper Check Refund";
+"payment_cancellation" = "Payment Cancellation";
+"payment_expiration" = "Payment Expired";
+"payment_expiry_fee" = "Payment Expiry Fee";
+"payment_fee" = "Payment Fee";
+"payment_return" = "Payment Return";
+"payment_reversal_fee" = "Payment Reversal Fee";
+"payment_reversal" = "Payment Reversal";
+"payment" = "Payment";
+"prepaid_card_balance_inquiry_fee" = "Balance Inquiry Fee";
+"prepaid_card_cash_advance" = "Cash Advance";
+"prepaid_card_dispute_deposit" = "Disputed Card Deposit";
+"prepaid_card_disputed_charge_refund" = "Disputed Charge Refund";
+"prepaid_card_domestic_cash_withdrawal_fee" = "ATM Withdrawal";
+"prepaid_card_exchange_rate_difference";
```

Comment:
```prepaid_card_exchange_rate_difference``` does not have locale?

Reply:
See the ticket, this PREPAID_CARD_EXCHANGE_RATE_DIFFERENCE is not provided, I will approve this pr and we can add this translation later on.